x/config/config.go

```diff
@@ -0,0 +1,117 @@
+package config
```

Comment:
Until UberFX goes 1.0, it would be nice to keep this section malleable, that way, we can make breaking changes if it's for the greater good.

We've been trying to release all our new features this way if possible:

1.  Feature lands in an `x/` package and gets released in a new minor 1.x
2. We partener with some early customers in production to iterate and test the feature
3. The feature gets promoted out of `x/`, where we commit to the API for the duration of 1.x

@alsamylkin does that make sense?